pkg/wasp/shortage-detector/shortage_detector.go

```diff
@@ -74,9 +86,19 @@ func (sdi *ShortageDetectorImpl) ShouldEvict() (bool, error) {
 		log.Log.Infof(fmt.Sprintf("Debug: averageSwapInPerSecond: %v condition: %v", averageSwapInPerSecond, averageSwapInPerSecond > sdi.maxAverageSwapInPagesPerSecond))
 		log.Log.Infof(fmt.Sprintf("Debug: averageSwapOutPerSecond:%v condition: %v", averageSwapOutPerSecond, averageSwapOutPerSecond > sdi.maxAverageSwapOutPagesPerSecond))
 	}
-	if overCommitmentRatioCondition {
-		log.Log.Infof("overCommitmentRatioCondition is true")
-		log.Log.Infof(fmt.Sprintf("Debug: overcommitment size:%v condition: %v", firstStat.SwapUsedBytes-firstStat.AvailableMemoryBytes-firstStat.InactiveFileBytes, overCommitmentRatioCondition))
-	}
-	return highTrafficCondition || overCommitmentRatioCondition, nil
```

Comment:
How about adding some logs if the highSwapUtilization condition met

Reply:
Done